strapi/.env.example

```
@@ -1,5 +1,6 @@
 HOST=0.0.0.0
 PORT=8001
+URL=http://localhost:3000/strapi
```

I think this should be the case if we now use port 80?
```suggestion
URL=http://localhost/strapi
```